### DISCUSSION TITLE
xformers

### DISCUSSION THREAD

**Discussion by nagaisa** [acheong08/Diffusion-ColabUI#38] [Ideas]
i see that xforms is disabled due to taking to long, from what i can tell adding the unsafe pickle arg can fix this so perhaps having it add that arg as well when it's on could be a fix

**Comment by acheong08** [comment]
The issue with xformers is that it is not installed on Google Colab by default and attempting to build it manually takes ~1hour.

**Reply by acheong08** [reply]
I did get it to install once but the improvements are miniscule (~1-3 second difference per image)
